.gitignore

```diff
@@ -30,3 +30,4 @@ yarn-error.log*
 .eslintcache
 .vscode
 .history
+.idea
```

Comment:
Others may use intellij as I do.

Reply:
Great, interllij is great, I used it when I was using Java. :D